opencsp/app/sofast/lib/CalibrateDisplayShape.py

```diff
@@ -335,7 +336,20 @@ def get_data(self) -> dict:
             self.data_calculation.pts_xyz_screen_aligned.data[:, self.data_calculation.intersection_points_mask]
         )
 
-        return {'pts_xy_screen_fraction': pts_xy_screen_fraction, 'pts_xyz_screen_coords': pts_xyz_screen}
+        return {'xy_screen_fraction': pts_xy_screen_fraction, 'xyz_screen_coords': pts_xyz_screen}
+
+    def as_DisplayShape(self, name: str) -> DisplayShape:
+        """Saves data to DisplayShape hdf file using the distorted3d model, see
```

Comment:
This doesn't actually save the data, does it? It just returns a DisplayShape instance?

Reply:
Nope, that was a copy/paste error.